code/modules/mob/living/simple_animal/hostile/gorilla.dm

```diff
@@ -0,0 +1,85 @@
+/mob/living/simple_animal/hostile/gorilla
+	name = "Gorilla"
+	desc = "A ground-dwelling, predominantly herbivorous ape that inhabits the forests of central Africa."
+	icon_state = "gorilla"
+	icon_living = "gorilla"
+	icon_dead = "gorilla_dead"
+	speak_chance = 80
+	maxHealth = 220
+	health = 220
+	loot = list(/obj/effect/gibspawner/generic)
+	butcher_results = list(/obj/item/weapon/reagent_containers/food/snacks/meat/slab/gorilla = 4)
+	response_help  = "prods"
+	response_disarm = "challenges"
+	response_harm   = "thumps"
+	speed = 1
+	melee_damage_lower = 15
+	melee_damage_upper = 18
+	damage_coeff = list(BRUTE = 1, BURN = 1.5, TOX = 1.5, CLONE = 0, STAMINA = 0, OXY = 1.5)
+	obj_damage = 20
+	environment_smash = 2
+	attacktext = "pummels"
+	attack_sound = 'sound/weapons/punch1.ogg'
+	faction = list("jungle")
+	robust_searching = TRUE
+	stat_attack = UNCONSCIOUS
+	minbodytemp = 270
+	maxbodytemp = 350
+
+	proc/target_bodyparts(atom/the_target)
```

Comment:
👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 
🔫 PUT THE RELATIVE PATHING DOWN
👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 👮 👮